module-federation-with-rspack/frontend/src/pluginRuntime.ts

```diff
@@ -0,0 +1,44 @@
+import { init, loadRemote } from "@module-federation/enhanced/runtime";
+import React from "react";
+import ReactDOM from "react-dom";
+import { PersesPluginModule } from "./PersesPlugin.types";
+
+init({
+  name: "@perses/perses-ui-host",
+  remotes: [
```

Comment:
I dont know how this `init` works but is it really dynamic here? Doesn't providing a given list of remotes here prevent you to "discover" other remotes later at runtime?